prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -945,13 +995,66 @@ class GetMultipleSettings : SettingCommand(
         }
 
         // get senders and receivers per org
-        val deepOrganizations = organizations.map { org ->
+        return organizations.map { org ->
             val sendersJson = getMany(environment, accessToken, SettingType.SENDER, org.name)
             val orgSenders = jsonMapper.readValue(sendersJson, Array<Sender>::class.java).map { it.makeCopy() }
             val receiversJson = getMany(environment, accessToken, SettingType.RECEIVER, org.name)
             val orgReceivers = jsonMapper.readValue(receiversJson, Array<ReceiverAPI>::class.java).map { Receiver(it) }
             DeepOrganization(org, orgSenders, orgReceivers)
         }
-        return yamlMapper.writeValueAsString(deepOrganizations)
+    }
+
+    private fun loadSettings(settings: List<DeepOrganization>) {
```

Comment:
Could you update this to something like `loadLocalSettings`? Need to be clear that some settings are being overriden.

Reply:
I find loadLocalSettings to be confusing because where are you loading the local settings to?